README.md

```diff
@@ -0,0 +1,84 @@
+
+### Local Deployments
+Local deployments make no online changes. See the `Running` section above for how to run the services yourself.
+
+### Pull Request Apps
+Upon making a pull request (draft or otherwise), a Heroku [Review App](https://devcenter.heroku.com/articles/github-integration-review-apps) will be created. On your PR, you'll see two buttons that say `View deployment`. These will open the two review apps (one for the CIS, one for the City) that get created with every new PR. The review apps are helpful because they allow pull request reviewers to have a visual way of seeing what parts of the API have changed (although reviewing the OpenAPI specification file is also good practice).
+
+Review app URLs look like `beacon-cis-pr-15.herokuapp.com` and `beacon-city-pr-15.herokuapp.com`, for example, if they were built from PR #15.
+
+### The QA Environment
+Our "QA" environment is just the `staging` Heroku apps in our backend pipelines. The `staging` app URLs are the following.
+- https://beacon-cis-main-staging.herokuapp.com/
+- https://beacon-city-main-staging.herokuapp.com/
+
+The staging apps are representative of the current state of the `main` branch of this repository. They are updated whenever there are any changes on that branch (like if a PR is merged).
+
+### Production
+Our production environments are the following URLs.
+- https://beacon-cis.herokuapp.com/
+- https://beacon-city.herokuapp.com/
+
+The production apps are whatever we choose to promote from `main` to the "production" environment. This happens with each point in time that we deem the API to be complete and stable.
+
+## OpenAPI Specifications / API Documentation
+The API for the backend services follow a documentation-first approach. The OpenAPI specifications for each service are used at build time to generate Java interfaces that are to be implemented in the codebase. This keeps the code behavior consistent with what the API documentation displays. The [CIS](common/src/main/openapi/beacon-central-identity-server.openapi.yml) spec may be found here and the City spec may be found [here](common/src/main/openapi/beacon-city.openapi.yml).
+
+If you want to view the _rendered_ version of the specification, you may do so by visiting the URL applicable to your current deployment scenario. If you're trying to view the changes you made locally, you'd visit the URL your service is running on (http://localhost:9876 for the CIS, http://localhost:8080 for the City). Note that if you make any changes to the spec, you'll need to restart the service so the Gradle `staticallyServeOpenAPISpec` task will be run again. You're in the right place if you see a web page with a layout like the image below.
+![image](https://user-images.githubusercontent.com/6992149/156421397-d951c851-e116-46c8-ac19-3c43cd3f4d21.png)
+
+## Manually Interacting with the Services
```

Comment:
I should change this section to just "Interacting with the Services" and make notes about (and links to) the NPM libraries that the frontend uses.